Pokedex/ViewController.swift

```diff
@@ -0,0 +1,345 @@
+//
+//  ViewController.swift
+//  Pokedex
+//
+//  Created by SAMEER SURESH on 9/25/16.
+//  Copyright © 2016 trainingprogram. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    let pokeGen = PokemonGenerator.getPokemonArray()
+    var pokemon : [Pokemon]! = []
+    var nameButton : UIButton!
+    var numButton : UIButton!
+    var attackButton : UIButton!
+    var healthButton : UIButton!
+    var defenseButton : UIButton!
+    var randomButton : UIButton!
+    var collectionView : UICollectionView!
+    var selectCategoryLabel : UILabel!
+    var searchBar : UITextField!
+    var byTypesLabel : UILabel!
+    var searchByCategoriesButton : UIButton!
+    var searchByTypesButton : UIButton!
+    var pressed : UIButton!
```

Comment:
need to initialize pressed, otherwise will run into an error when categoreisHooHah is clicked without a "pressed" value